### PR TITLE
add users and roles section - TEDU-130

### PR DESCRIPTION
**Pull request by favour-chibueze**
TEDU-130

<a href="https://gitpod.io/#https://github.com/mautic/user-documentation/pull/108"></a>

### CODE REVIEW

**Comment by RCheesley** [comment]
Please fix the Vale issues with the PR :)

File: docs/users_roles/managing_roles.rst [hunk 1]
```diff
@@ -1,6 +1,105 @@
...
+
+* **Specified fields** - allow or deny the Users with this Role to edit specified fields in the Users section for example, Name, Username, Email, Position.
+
+* **All** - this allows the Users with this Role to edit all fields relating to the Users section
+
+.. note:: 
+
+  User permissions restrict their view of dashboard widgets, resulting in them only seeing widgets for items or feature bundles they have permission to see. 
+  
+  For example, if a User’s Role doesn’t have Asset permissions, they can’t create widgets on the dashboard for Asset data.
```
Comment (RCheesley) [inline docs/users_roles/managing_roles.rst:103]:
```suggestion
  For example, if a User’s Role doesn’t have Asset permissions, they can’t create or view widgets on the dashboard for Asset data.
```

File: docs/users_roles/managing_users.rst [hunk 2]
```diff
@@ -3,4 +3,41 @@
...
+  
+.. note:: 
+
+    Passwords must be at least six (6) characters in length. Ensure that you use a combination of upper and lower case alphabets, special characters, and numbers.
+
+* **Time zone** - Set the User’s time zone, or use the default. Adding the User’s time zone enables them to account for time zone differences for Email scheduling and other features.
+
+* **Language** - Select a language for each User, to improve their experience in Mautic.
+
+Notify your Users that their accounts have been created. Mautic doesn’t send an Email notifying Users of their login information. Manually informing them is necessary.
```
Comment (RCheesley) [inline docs/users_roles/managing_users.rst:43]: We have a PR to change this - hopefully it makes it into Mautic 5! https://github.com/mautic/mautic/pull/9733

**Review by RCheesley — CHANGES_REQUESTED** [review]
Tiny suggestion then good to go!

File: docs/channels/utm_tags.rst [hunk 3]
```diff
@@ -39,12 +39,12 @@ The following table lists the Google Analytics UTM tags:
      - Description
    * - Campaign Source
      - The referring source of the web activity. It indicates the social network, search engine, newsletter name, or any other specific source driving the traffic.  
-        * Examples: facebook, twitter, blog, newsletters
+        * Examples: ``facebook``, twitter, blog, newsletters
```
Comment (RCheesley) [inline docs/channels/utm_tags.rst:42]: Why is this one commented and the others not? Let's do all or none.

File: docs/channels/utm_tags.rst [hunk 4]
```diff
@@ -39,12 +39,12 @@ The following table lists the Google Analytics UTM tags:
...
         * UTM code: utm_source
-        * Sample code: utm_source=facebook
+        * Sample code: ``utm_source=facebook``
    * - Campaign Medium
      - UTM tags - Mautic Documentation
-         * Examples: cpc, organic_social
+         * Examples: ``cpc``, organic_social
```
Comment (RCheesley) [inline docs/channels/utm_tags.rst:47]: As above - why just this one commented and the rest not?

**Comment by RCheesley** [comment]
@favour-chibueze this PR also includes UTM tags and some settings changes - any reason why? I will review the whole PR but maybe update the title.

File: docs/users_roles/managing_roles.rst [hunk 5]
```diff
@@ -1,6 +1,105 @@
...
-Managing Roles
-##############
+Roles
+#####
 
-.. vale on
\ No newline at end of file
+.. vale on
+
+Access to Mautic instances is controlled by creating accounts for Users and associating them with a Role.
```
Comment (RCheesley) [inline docs/users_roles/managing_roles.rst:8]:
```suggestion
You can control access to Mautic instances by creating accounts for Users and associating them with a Role.
```

Vale fixes

File: docs/users_roles/managing_roles.rst [hunk 6]
```diff
@@ -1,6 +1,105 @@
...
+
+* **Own** - this allows the Users with this Role to ``view/edit/delete/publish`` their own resources in this part of Mautic, but not those created by others
+
+* **Others** - this allows the Users with this Role to ``view/edit/delete/publish`` their own resources in this part of Mautic, and those created by others
+
+There are permission levels relating to being able to manage resources:
+
+* **Manage** - this allows the Users with this Role to manage resources in this area of Mautic for example, managing custom fields or Plugins.
+
+There are permission levels relating to what fields in the Users section can be edited:
```
Comment (RCheesley) [inline docs/users_roles/managing_roles.rst:93]:
```suggestion
There are permission levels relating to the editable fields in the Users section:
```

File: docs/users_roles/managing_roles.rst [hunk 7]
```diff
@@ -1,6 +1,105 @@
...
+
+* **Specified fields** - allow or deny the Users with this Role to edit specified fields in the Users section for example, Name, Username, Email, Position.
+
+* **All** - this allows the Users with this Role to edit all fields relating to the Users section
+
+.. note:: 
+
+  User permissions restrict their view of dashboard widgets, resulting in them only seeing widgets for items or feature bundles they have permission to see. 
+  
+  For example, if a User’s Role doesn’t have Asset permissions, they can’t create or view widgets on the dashboard for Asset data.
```
Comment (RCheesley) [inline docs/users_roles/managing_roles.rst:103]:
```suggestion
  For example, if a User's Role doesn't have Asset permissions, they can't create or view widgets on the dashboard for Asset data.
```

File: docs/users_roles/managing_users.rst [hunk 8]
```diff
@@ -3,4 +3,41 @@
...
+
+To set up a User manually:
+
+1. Navigate to **Settings** > **Users**.
+
+2. Click **+New** in the top right corner.
+
+3. Fill in the appropriate fields for your User:
+   
+   * **First name** and **Last name** - Your User’s first and last name.
```
Comment (RCheesley) [inline docs/users_roles/managing_users.rst:25]:
```suggestion
   * **First name** and **Last name** - Your User's first and last name.
```

File: docs/users_roles/managing_users.rst [hunk 9]
```diff
@@ -3,4 +3,41 @@
...
+
+   * **Roles** are assigned based on permissions you grant.
+  
+   * **Signature** is only necessary if you’re using the Mailer is owner feature in Mautic.
```
Comment (RCheesley) [inline docs/users_roles/managing_users.rst:29]:
```suggestion
   * **Signature** is only necessary if you're using the Mailer is owner feature in Mautic. 
```

File: docs/users_roles/managing_users.rst [hunk 10]
```diff
@@ -3,4 +3,41 @@
...
+
+3. Fill in the appropriate fields for your User:
+   
+   * **First name** and **Last name** - Your User’s first and last name.
+
+   * **Roles** are assigned based on permissions you grant.
+  
+   * **Signature** is only necessary if you’re using the Mailer is owner feature in Mautic. 
+
+   * **Position (optional)** - Your User’s job title.
```
Comment (RCheesley) [inline docs/users_roles/managing_users.rst:31]:
```suggestion
   * **Position - optional** - Your User's job title.
```

File: docs/users_roles/managing_users.rst [hunk 11]
```diff
@@ -3,4 +3,41 @@
...
+
+1. Navigate to **Settings** > **Users**.
+
+2. Click **+New** in the top right corner.
+
+3. Fill in the appropriate fields for your User:
+   
+   * **First name** and **Last name** - Your User’s first and last name.
+
+   * **Roles** are assigned based on permissions you grant.
```
Comment (RCheesley) [inline docs/users_roles/managing_users.rst:27]:
```suggestion
   * **Roles** assigned based on permissions you grant.
```

File: docs/users_roles/managing_users.rst [hunk 12]
```diff
@@ -3,4 +3,41 @@
...
+  
+   * **Signature** is only necessary if you’re using the Mailer is owner feature in Mautic. 
+
+   * **Position (optional)** - Your User’s job title.
+  
+   * **Username, Email, Password** are the login credentials. If a User forgets their password, they can use the Forgot password link, but you can manually change their password here.
+  
+.. note:: 
+
+    Passwords must be at least six (6) characters in length. Ensure that you use a combination of upper and lower case alphabets, special characters, and numbers.
```
Comment (RCheesley) [inline docs/users_roles/managing_users.rst:37]:
```suggestion
    Passwords must be at least six characters in length. Ensure that you use a combination of upper and lower case alphabets, special characters, and numbers.
```

I don't think we need to spell out the number here.

File: docs/users_roles/managing_users.rst [hunk 13]
```diff
@@ -3,4 +3,41 @@
...
+
+* **Time zone** - Set the User’s time zone, or use the default. Adding the User’s time zone enables them to account for time zone differences for Email scheduling and other features.
```
Comment (RCheesley) [inline docs/users_roles/managing_users.rst:39]:
```suggestion
* **Time zone** - Set the User's time zone, or use the default. Adding the User's time zone enables them to account for time zone differences for Email scheduling and other features.
```

File: docs/users_roles/managing_users.rst (hunk 2 above)
Comment (RCheesley) [inline docs/users_roles/managing_users.rst:43]:
```suggestion
Notify your Users that their accounts have been created. Mautic doesn't send an Email notifying Users of their login information. Manually informing them is necessary.
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
docs/users_roles/images/full-access-roles.png needs to be updated - the save buttons are the wrong way round.

A bunch of Vale fixes the same as the other PRs.

Please look for this in VSCode when it comes to copy/pasted text and apostrophes:

screenshot-favourchibu-userdocumen-sjdx6d2n0qz ws-eu78 gitpod io-2022 12 09-11_50_32

To fix, delete and re-type the apostrophe.

File: docs/channels/utm_tags.rst (hunk 4 above)
Comment (favour-chibueze) [reply docs/channels/utm_tags.rst:47]: it was throwing a vale error but I will comment both instead of having one commented

**Comment by favour-chibueze** [comment]
@RCheesley

**Review by RCheesley — APPROVED** [review]
There was a small Vale fix needed on the last line which I've pushed rather than go through another round of review!

Thanks for this, LGTM! 🚀

**Pull request merged**